### DISCUSSION TITLE
Hasura CLI picking up non-existing version

### DISCUSSION THREAD

**Discussion by ZilvinasAbr** [hasura/graphql-engine#9302] [Question] [answered]
Hi, our project on github is running this script:
```bash
curl -L https://github.com/hasura/graphql-engine/raw/stable/cli/get.sh | bash
```

For some reason it tries to fetch the graphql-engine version 2.16.1 from github releases, which does not yet exist.

We are blocked from building our application in Github Pull requests

**Comment by ZilvinasAbr** [comment]
**Finding:** Our configs use older Hasura Graphql engine version: `hasura/graphql-engine:v2.2.0`. We will try to update it and see if the issue gets resolved

**Reply by ZilvinasAbr** [reply]
Updating the version did not help to resolve the issue

**Comment by povilasDadelo** [comment]
I use `2.15.1` but the CI in github is failing because it is trying to get `2.16.1`

**Comment by ZilvinasAbr** [comment, marked as answer]
Resolved the issues, the issue was that the `v2.16.1` assets where not published yet
